source/_components/light.iglo.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: "iGlo"
+description: "Instructions how to integrate LIFX into Home Assistant."
+date: 2017-12-17 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: iglo.png
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.60
```

Comment:
This certainly would not make it into the `0.60` release, since it it already out there.
You might want to consider changing it to `0.61`.